increment/src/lib.rs

```diff
@@ -8,10 +8,19 @@ pub struct IncrementContract;
 
 #[contractimpl]
 impl IncrementContract {
-    /// Increment increments an internal counter, and returns the value.
+    /// Initialize the counter with a value
+    pub fn __constructor(env: Env, counter: u32) {
```

Comment:
This will break anyone relying on this contract for testing since now they need to call init, right?

Reply:
Yes but I think it’s only the JS sdk for now

Reply:
And you need to opt in to newer commits, if you’re using submodules

Reply:
fair, lgtm then :+1: